doc/nrf/known_issues.rst

```diff
@@ -1533,7 +1552,14 @@ The combination of nRF Secure Immutable Bootloader and MCUboot fails to upgrade
 Build system
 ============
 
-.. rst-class:: v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0
+.. rst-class:: v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0
```

Comment:
Is the nanopb issue valid for all releases since 1.3.0? @peknis , @tejlmand ?

Reply:
no, it's only a valid issue from v2.0.0, cause before that nanopb - protoc was not part of the toolchain and uesrs at that time were requested to install this tool manually.

Reply:
```suggestion
.. rst-class:: v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0
```